EthereumKit/Networking/Request/JSONRPC.swift

```diff
@@ -80,4 +80,45 @@ public final class JSONRPC {
             return Response(id: transactionID)
         }
     }
+    
+    public struct Call: JSONRPCKit.Request {
+        public typealias Response = String
+        
+        public let from: Address?
+        public let to: Address
+        public let gas: Int?
+        public let gasPrice: Int?
+        public let value: Int?
```

Comment:
Use `Gas.GasLimit` for gas and `Gas.GasPrice` for gas price instead 👍

Reply:
I got it 🙏!